packages/amplication-client/src/ModuleActions/ModuleActions.tsx

```diff
@@ -87,68 +100,167 @@ const ModuleActions = React.memo(({ match }: Props) => {
     data?.serviceSettings?.serverSettings?.generateGraphQL;
 
   return (
-    <>
-      <>
-        <div className="module-toggle-field__search-field">
-          <SearchField
-            label="search"
-            placeholder="Search"
-            onChange={handleSearchChange}
-          />
-        </div>
-
-        <FlexItem
-          itemsAlign={EnumItemsAlign.Center}
-          margin={EnumFlexItemMargin.Top}
-          start={
-            <TabContentTitle
-              title="Module Actions"
-              subTitle="Actions are used to perform operations on resources, with or without API endpoints."
-            />
-          }
-          // end={<NewModuleAction resourceId={resourceId} moduleId={moduleId} />} todo: return in phase 2
-        ></FlexItem>
-
-        {generateGraphQlAndRestApi && (
-          <FlexItem
-            direction={EnumFlexDirection.Row}
-            contentAlign={EnumContentAlign.Start}
-            itemsAlign={EnumItemsAlign.Normal}
-          >
-            GraphQL API
-            <div className={`module-toggle-field__operation-toggle`}>
-              <Toggle
-                checked={displayMode === EnumApiOperationTagStyle.REST}
-                onValueChange={handleDisplayModeChange}
-              />
-            </div>
-            REST API
-          </FlexItem>
-        )}
-
-        {moduleId ? (
-          <FlexItem margin={EnumFlexItemMargin.Top}>
-            <ModuleActionList
-              moduleId={moduleId}
-              resourceId={resourceId}
-              searchPhrase={searchPhrase}
-              displayMode={displayMode}
-            />
-          </FlexItem>
-        ) : (
-          moduleListData?.Modules.map((module) => (
-            <FlexItem key={module.id} margin={EnumFlexItemMargin.Top}>
+    <FeatureIndicatorContainer
```

Comment:
do you see how it can be under a simple prop. this is the reason I wanted to remove this

Reply:
please take those prop and build it as a separate components (for disabled and not) it will be more readable